openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/TestSummary.style.less

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@import url('../../../styles/variables.less');
+
+.parameter-value-container {
+  background-color: @test-parameter-bg-color;
+  border: 1px solid @border-color;
+  border-radius: 4px;
+  padding: 2px;
+}
+
+.sql-expression-container {
+  .table-query-editor {
+    .CodeMirror {
+      border: 1px solid @border-color;
+      border-radius: 4px;
+    }
+    .CodeMirror-gutters {
+      border: none;
+    }
+    .CodeMirror-linenumber {
+      color: @text-color;
+    }
+  }
+}
```

Comment:
can you check code-mirror.less, we can use that instead of declaring new new style